Unable to start service because of FlutterBleLib

You used       
 ```
final FlutterBleLibPlugin plugin = new FlutterBleLibPlugin(registrar.activity().getApplicationContext());
 ```
so when i start service from a broadcast (in my case geolocation via geofencing) i do:
 ```
sBackgroundFlutterView = FlutterNativeView(context, true)
val registry = sBackgroundFlutterView!!.pluginRegistry
sPluginRegistrantCallback.registerWith(registry)
```
but this cause a NullPointerException because activity is null

there 's a bettery way to create the plugin? or a better way to load plugins from service?

still null
`     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.Context android.app.Activity.getApplicationContext()' on a null object reference
`

--- a/android/src/main/java/com/polidea/flutter_ble_lib/FlutterBleLibPlugin.java
+++ b/android/src/main/java/com/polidea/flutter_ble_lib/FlutterBleLibPlugin.java
@@ -61,7 +61,7 @@ public class FlutterBleLibPlugin implements MethodCallHandler {
         final EventChannel connectionStateChannel = new EventChannel(registrar.messenger(), ChannelName.CONNECTION_STATE_CHANGE_EVENTS);
         final EventChannel characteristicMonitorChannel = new EventChannel(registrar.messenger(), ChannelName.MONITOR_CHARACTERISTIC);
 
-        final FlutterBleLibPlugin plugin = new FlutterBleLibPlugin(registrar.activity().getApplicationContext());
+        final FlutterBleLibPlugin plugin = new FlutterBleLibPlugin(registrar.context());
 
         channel.setMethodCallHandler(plugin);
 

